gensim/sklearn_api/tfidf.py

```diff
@@ -60,31 +60,37 @@ def __init__(self, id2word=None, dictionary=None, wlocal=gensim.utils.identity,
             The mnemonic for representing a combination of weights takes the form XYZ,
             for example 'ntc', 'bpn' and so on, where the letters represents the term weighting of the document vector.
 
-            Term frequency weighing:
-                * `n` - natural,
-                * `l` - logarithm,
-                * `a` - augmented,
-                * `b` - boolean,
-                * `L` - log average.
-
-            Document frequency weighting:
-                * `n` - none,
-                * `t` - idf,
-                * `p` - prob idf.
-
-            Document normalization:
-                * `n` - none,
-                * `c` - cosine.
+            local_letter : str
```

Comment:
Can we hyperlink to resolve_weights instead?

Reply:
See above.